test/color/color_dump.c

```diff
@@ -34,6 +34,7 @@
 
 static struct ConfigDef Vars[] = {
   // clang-format off
+  { "attachments_respect_umask", DT_BOOL, false, 0, NULL, },
```

Comment:
Wow, I never would have guessed that, thanks!

Reply:
hehe, it's a temporary hack.

You added a config lookup in `mutt_file_open()`.
 
```c
if (cs_subset_bool(NeoMutt->sub, "attachments_respect_umask"))
```

The `mutt/` library shouldn't have any dependencies on other libraries.
We can fix that later.

Reply:
> hehe, it's a temporary hack.
> 
> You added a config lookup in `mutt_file_open()`.
> 
> ```c
> if (cs_subset_bool(NeoMutt->sub, "attachments_respect_umask"))
> ```
> 
> The `mutt/` library shouldn't have any dependencies on other libraries. We can fix that later.

I don't see the full picture yet, so I'll let it go :).